src/Symfony/Component/HttpClient/Tests/NativeHttpClientTest.php

```diff
@@ -48,4 +50,25 @@ public function testHttp2PushVulcainWithUnusedResponse()
     {
         $this->markTestSkipped('NativeHttpClient doesn\'t support HTTP/2.');
     }
+
+    public function testIPv6Resolve()
+    {
+        TestHttpServer::start(-8087);
+
+        DnsMock::withMockedHosts([
+            'symfony.com' => [
+                [
+                    'type' => 'AAAA',
+                    'ipv6' => '::1',
+                ],
+            ],
+        ]);
+
+        $client = $this->getHttpClient(__FUNCTION__);
+        $response = $client->request('GET', 'http://symfony.com:8087/');
+
+        $this->assertSame(200, $response->getStatusCode());
+
+        DnsMock::withMockedHosts([]);
```

Comment:
that line is not needed

Reply:
Having those mocked DNS functions registered and some hosts overrides defined affects other parent `HttpClientTest` tests.
Removing this line, which effectively clears the overrides make those tests pass.
I can investigate a bit more, why they fail.

Reply:
<img width="1254" alt="image" src="https://github.com/user-attachments/assets/45a51b78-c6e8-488a-88db-604a19fab1d1">